compiler/env/VerboseLog.cpp

```diff
@@ -63,18 +63,19 @@ const char * TR_VerboseLog::_vlogTable[] =
    "#PROFILING: ",
    "#JITServer: ",
    "#AOTCOMPRESSION: ",
+   "FSD: ",
    };
 
 void TR_VerboseLog::writeLine(TR_VlogTag tag, const char *format, ...)
    {
-   TR_ASSERT(tag != TR_Vlog_null, "TR_Vlog_null is not a valid Vlog tag");
-   va_list args;
-   va_start(args, format);
-   writeTimeStamp();
-   write(_vlogTable[tag]);
-   vwrite(format,args);
+   write(tag, format);
+   write("\n");
+   }
+
+void TR_VerboseLog::writeLine(const char *format, ...)
+   {
+   write(format);
    write("\n");
-   va_end(args);
    }
 
 void TR_VerboseLog::writeLineLocked(TR_VlogTag tag, const char *format, ...)
```

Comment:
Nitpick (feel free to ignore): since you're refactoring, might be worth having `writeLineLocked` just call `writeLine` after acquiring the lock.

Reply:
As far as I can tell there is no easy way to do handoff for variadic functions in C [1]. The only way this will work is if we convert these APIs into variadic template functions in C++. Not sure if those are supported on all our compiler toolchains. @Leonardo2718 do you might know?

I think this is probably where we want to go but I would propose this be done outside the scope of this PR if possible to reduce potential of bugs.

[1] http://c-faq.com/varargs/handoff.html

Reply:
Ah sorry, I made the comment in the context of the previous changes when `writeLine` was calling `write`; now that it's calling `vwrite` it doesn't apply.